python/ray/rllib/agents/agent.py

```diff
@@ -137,6 +137,10 @@
     # Similar to remote_worker_envs, but runs the envs asynchronously in the
     # background for greater efficiency. Conflicts with remote_worker_envs.
     "async_remote_worker_envs": False,
+    # Number of last seen checkpoints to keep on disk
+    "keep_checkpoints": 30,
```

Comment:
move these to tune, where `checkpoint_freq` is. Follow the path that argument is going. Your arguments should be able to be used from CLI as well.